package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "test": "nodeunit test.js"
+    "test": "./node_modules/.bin/nodeunit test.js"
```

Comment:
`npm run` puts the `.bin` early in the path, so if you did an `npm install` in this repo dir, `npm test` should use the version of `nodemon` that's specified in `devDepenencides`.